src/libstore/globals.hh

```diff
@@ -860,10 +860,56 @@ public:
         )"};
 
     Setting<std::string> githubAccessToken{this, "", "github-access-token",
-        "GitHub access token to get access to GitHub data through the GitHub API for `github:<..>` flakes."};
+        "GitHub access token to get access to GitHub data through the GitHub API for `github:<..>` flakes (deprecated, please use 'access-tokens' instead)."};
```

Comment:
We can just remove this option.

Reply:
Sounds good, I've removed this config option, @edolstra .